LIC_FILES_CHKSUM.sha256

```diff
@@ -40,6 +40,7 @@ a2e6011b982212228008c4386616a4818eb5fdf9ea5805b7a90500e6ce895aaf  vendor/github.
 95b8ef9c4137a8f75ddd3101ffdc4cfd594fa875b261697b68baddc16b0e537c  vendor/github.com/konsorten/go-windows-terminal-sequences/LICENSE
 65bffcb78f7d0f57b2e18dcd728d398a86a21832adb61a25c35cc7363e107d19  vendor/github.com/ugorji/go/LICENSE
 9e321f6a1db81616a6c344d3aefddf9834fef23bdd913eae0db283dfa6753dff  vendor/gopkg.in/go-playground/validator.v8/LICENSE
+6baef92671abea7e5b10bb50f691ee8329422baf46ef6cd5bf29928bf38adee1  vendor/github.com/thedevsaddam/gojsonq/LICENSE.md
```

Comment:
general comment here, do we need in the pushed vendor tree all those files: .gitignore, .travis, etc?

Reply:
No idea, I failed to use govendor to, ehm, vendor the package.
Any suggestion on how to properly vendor this thing? For my own projects I use go modules, sooooo eaaaasy.

Reply:
```bash
# govendor fetch vendor/github.com/thedevsaddam/gojsonq
# govendor sync +all
```
we need to migrate to modules so bad -- agreed.

Reply:
sorry, my mistake, fetch like this:
```bash
# govendor fetch github.com/thedevsaddam/gojsonq
```

Reply:
Done